run_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Copyright (C) 2016 Swift Navigation Inc.
+# Contact: Mark Fine <mark@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+#
+# Script for running developer tests.
+
+set -e
+
+mkdir -p package/build && cd package/build
+cmake ../ && make
```

Comment:
I think once there are tests implemented, we should add tests as a default target to the cmake.